lambda-views/macros/lambda/lambda_union.sql

```diff
@@ -0,0 +1,37 @@
+{% macro lambda_union(historical_relation, model_sql) %}
+
+{% set unique_key = config.get('unique_key', none) %}
+
+with historical as (
+    
+    select * from {{ historical_relation }}
+    
+),
+
+new as (
+    
+    {{ model_sql }}
```

Comment:
oh coool so it's only one additional model? Smart! Might change how we show this in a diagram though!

Reply:
Yeah! This is how @amychen1776 implemented previously. Fewer model files and database objects floating around. Agree that the conceptual diagram is a _little_ less clear as a result, though. FWIW we could consider an ephemeral model here for DAG clarity.

Reply:
Yup! That was the initial intention. However, because it was confusing I opted to do the less dry method to make it more approachable. I think we should show this approach but not update the diagram to keep the concept very straight to the point.